model/models.js

```diff
@@ -12,9 +12,38 @@ exports.fetchArticleById = (article_id) => {
       if (!article) {
         return Promise.reject({
           status: 404,
-          msg: `No article found for article_id ${article_id}`,
+          message: `No article found for article_id ${article_id}`,
         });
       }
       return article;
     });
 };
+
+exports.fetchAllArticles = () => {
+  return db
+    .query(
+      `
+    SELECT
+      articles.article_id,
+      articles.author,
+      articles.title,
+      articles.topic,
+      articles.created_at,
+      articles.votes,
+      articles.article_img_url,
+      COUNT(comments.article_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    GROUP BY
+      articles.article_id,
```

Comment:
You only need one value for the `GROUP BY` so line `38` is enough on it's own for the grouping